lib/styles/components/auth-state.less

```diff
@@ -85,13 +85,13 @@
                 width: auto;
                 height: 100%;
                 text-align: center;
+                padding: 15px;
                 svg {
                     display: inline-block;
                     position: relative;
                     top: -1px;
                     width: auto;
                     height: inherit;
-                    padding: 30%;
```

Comment:
we prefer to stay flexible as much as possible.

Reply:
The use of percentage value is not appropriate. There is no absolute value (like pixel) of width provided in the ancestor element of svg.
I have removed this padding as it was causing issue in Firefox, and used 15px in a parent `<a>` tag which is what was being rendered previously (in Chrome).

See https://developer.mozilla.org/en-US/docs/Web/CSS/padding#Values for use of percentage value of padding.

Reply:
gotya, this is ok then